features/buyers/service_contracts/bulk_operations.feature

```diff
@@ -97,7 +97,7 @@ Feature: Service subscriptions bulk operations
     And should see "You have selected subscriptions to plans from different services"
 
   Scenario: Change state in bulk
-    Given the table should contain the following:
+    Given the table looks like:
```

Comment:
Does this change has anything to do with order? If order should match, then new name makes good sense. If order should not matter, then original step name speaks better.

Reply:
Looking at https://github.com/3scale/porta/pull/3663/files#diff-2212f1a688f04ba9915f0c91550ed4808748d2547115563b9d7dc3318e89064fR49
it makes sense, thank you.

Reply:
Seems like original name explains it better:
https://github.com/3scale/porta/pull/3663/files#diff-2212f1a688f04ba9915f0c91550ed4808748d2547115563b9d7dc3318e89064fR49